template/src/lib/service.broker.ts

```diff
@@ -0,0 +1,88 @@
+import { TypedServiceBroker } from 'moleculer-service-ts';
+import { Service, Middleware, ServiceBroker, EventSchema } from 'moleculer';
+import {
+  MikroConnector,
+  DatabaseContextManager,
+  MoleculerMikroContext
+} from 'moleculer-context-db';
+import { ServiceAction, ServiceEvent, ServiceName } from '../service.types';
+import { entities } from '../entities';
+import { config } from './env';
+
+import { brokerConfig } from './moleculer/broker.config';
+import { {{capitalizedServiceName}}Service } from '../{{serviceName}}.service';
+
+let service: Service;
+let started = false;
+
+export const broker: TypedServiceBroker<
+  ServiceAction,
+  ServiceEvent,
+  ServiceName
+> = new TypedServiceBroker(brokerConfig);
+
+export const dbConnector: MikroConnector = new MikroConnector();
```

Comment:
For ```mongo```, one needs to type this with
```ts
import { MongoDriver } from 'mikro-orm/dist/drivers/MongoDriver';

dbConnector: MikroConnector = new MikroConnector<MongoDriver>();
```

Since stuff in ```lib``` should not be changed, and different services will have different db needs, I think it is best to move all the db stuff out of ```lib``` to ```main.ts```

Reply:
I thought about this - can we use the same database across all services?
Why? - less complex and consistent code (only thing that came into my mind)